nomos-mix/core/src/message_blend/mod.rs

```diff
@@ -136,10 +36,15 @@ pub enum MessageBlendStreamOutgoingMessage {
     Outbound(Vec<u8>),
 }
 
+/// [`MessageBlendStream`] handles the entire mixing tiers process
```

Comment:
As commented in https://github.com/logos-co/nomos-node/pull/896/files#r1822516530, I think it's better to let `MessageBlendStream` handles only the tier 2 (not including tier 1), because Message Blend is the name of tier 2.

Reply:
Yes, I will extract again and try to abstract the Tiers themselves.